makefiles/check.mk

```diff
@@ -44,4 +44,4 @@ run-client-doc-test: install_rustup_target
 
 
 fluvio_run_bin: install_rustup_target
-	cargo build --bin fluvio-run $(RELEASE_FLAG) --target $(TARGET)
+	cargo build --bin fluvio-run --features wasi $(RELEASE_FLAG) --target $(TARGET)
```

Comment:
We don't want to build with this feature by default. I think that we can add here $(DEBUG_SMARTMODULE_FLAG) just like we do with `RELEASE_FLAG`.

We can define `DEBUG_SMARTMODULE_FLAG` in the Makefile in the root of the repo with something like 

```bash
DEBUG_SMARTMODULE_FLAG=$(if $(DEBUG_SMARTMODULE),--features wasi,)
```

The same applies to `makefiles/build.mk`

Reply:
Sorry, never dealt with Makefiles. Done now.